tests/Microsoft.Bot.Builder.Tests/chatdown_gen.cmd

```diff
@@ -0,0 +1,4 @@
+@ECHO OFF
```

Comment:
I'm not clear on why this file shows up twice. Once in the Core.Extensions.Tests project and once in the Bot.Builder.Tests project.

Reply:
We deleted others `chatdown_gen.cmd`